iot_verifier/src/loader.rs

```diff
@@ -54,6 +54,12 @@ pub enum NewLoaderError {
     DenyListError(#[from] denylist::Error),
 }
 
+pub enum ValidGatewayResult {
```

Comment:
It might be more readable to have this called `GatewayValidity`, and then return an `Option<GatewayValidity>` with the None matching unknown

Reply:
i gave this a try and tbh i think it is less readable.  Unknown here is an explicit result, ie this gateway has been checked and is not found.  It is not the absence of a result.